src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String LINE = "____________________________________________________________";
+    public static final int MAX = 100;
+
     public static void main(String[] args) {
```

Comment:
Can shift main function down to be at the very bottom, other functions be above it